build_runner/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 - Improved the layout of `/$perf`, especially after browser window resize.
 - `pub run build_runner` exits with a error when invoked with an unsupported
   command.
+- Bug Fix: Update outputs in merged directory for sources which are not used
```

Comment:
It might be nice to give an example, i.e.:

"during the build - a common example is changing web/index.html"

Reply:
Done